SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialConnectionManagerTest.cpp

```diff
@@ -118,7 +118,7 @@ CONNECTIONMANAGER_TEST(SetupFromURL_Receptionist_Localhost)
 {
 	// GIVEN
 	FTemporaryCommandLine TemporaryCommandLine("");
-	const FURL URL(nullptr, TEXT("127.0.0.1"), TRAVEL_Absolute);
+	const FURL URL(nullptr, TEXT("127.0.0.1:777"), TRAVEL_Absolute);
```

Comment:
```suggestion
	const FURL URL(nullptr, TEXT("127.0.0.1:7777"), TRAVEL_Absolute);
```
Nit 7777 is the actual runtime port

Reply:
Ah I see you're using a different one and then testing the default. Potentially ignore this then.